graph/RM6119_Furniture_and_Associated_Services.cypher

```diff
@@ -0,0 +1,264 @@
+CREATE
+
+// QUESTIONS
+
+// Questions: level 1
+
+(qstnFirstQ:Question {
+    uuid: 'e40d4dee-53fc-493b-a1d3-3e86d17add1e',
+    text: 'What Service or Product are you looking for?',
+    type: 'LIST'
+}),
+
+
+// Questions: level 2
+(qstnWhatProduct:Question {
+    uuid: '25e5d903-2f2a-4a55-9382-5fb935dab79d',
+    text: 'What product do you need?',
+    type: 'LIST'
+}),
+
+(qstnWhatService:Question {
+    uuid: '06a14aa8-7b90-4e2a-a249-a55e8458e644',
+    text: 'What service do you need?',
+    type: 'LIST'
+}),
+
+// (qstnRepairAndRenovation:Question {
+//     uuid: '5d55a804-361a-465b-a515-48a87688006e',
+//     text: 'Repair and Renovation',
+//     type: 'LIST'
+// }),
+
+
+// Questions: level 3
+(qstnTypeOfficeFurniture:Question {
+    uuid: '8f0e8b24-b61f-4b4b-9bc4-99197d9253ae',
+    text: 'What type of office furniture do you need?',
+    type: 'LIST'
+}),
+
+// Questions: level 4
+(qstnSpecificFurnitureNeeds:Question {
+    uuid: 'a765ee32-5f0b-4022-9f22-d1765d5cd6f9',
+    text: 'Do you have specific furniture needs for further or higher education?',
+    type: 'LIST'
+}),
+
+
+
+// ANSWERS
+
+// Answers: level 1
+(ansProduct:Answer {
+    uuid: '518afdfd-c077-41f5-842c-c42b35ad4c3b',
+    text: 'Product',
+    hint: 'A product is an item you can buy. It\'s a one-off purchase and involves no ongoing contract.'
```

Comment:
remove all the full stop at the end of the sentence for the hint text

Reply:
done